feature/my/src/main/java/com/hankki/feature/my/mypage/component/IconButton.kt

```diff
@@ -0,0 +1,62 @@
+package com.hankki.feature.my.mypage.component
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.size
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.hankki.core.common.extension.noRippleClickable
+import com.hankki.core.designsystem.theme.Gray600
+import com.hankki.core.designsystem.theme.HankkiTheme
+import com.hankki.core.designsystem.theme.HankkijogboTheme
+import com.hankki.feature.my.R
+
+@Composable
+fun IconButton(
+    image: Int,
```

Comment:
이것도 물론 위에 말씀드린 것 처럼 imageVector로 바꾸는게 좋겠지만, 만약 다른 프로젝트에서 image에 해당하는 Resource값을 받아온다면 이렇게 어노테이션으로 나타내주는게 좋습니다.
```suggestion
    @DrawableRes image: Int,
```